scripts/CsvCaseDataUpdater.java

```diff
@@ -0,0 +1,253 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.*;
+
+/**
+ * Takes a CSV file and is used to update the case values for specific cases.
+ *
+ * Format of the csvs must be as follows: {caseUUID, [fields]+}, with the first
+ * row being the column headers.
+ *
+ * Fields that hold no value within the CSV are ignored when it comes to updating.
+ *
+ * To use this file you first need to add the CSV file to the K8 hocs-casework
+ * container through {@code kubectl cp [CSVFilePath] [PodName]:. -c hocs-casework}.
+ * You can then invoke this java file through the containers command line by running
+ * {@code java CsvCaseDataUpdater.java [AbsoluteCSVFilePath]}.
+ */
+public class CsvCaseDataUpdater {
+
+    /*
+     * We inject this variable through deployment, so that the spring boot application knows what to run on.
+     * Default to the spring boot 8080, incase it's not present.
+     */
+    private static final String SERVER_PORT =
+            getEnvironmentVariableOrDefault("SERVER_PORT", "8080");
+    private static final String BASE_URL_FORMAT = "http://localhost:" + SERVER_PORT + "/case/%s/data/%s";
+
+    public static void main(String[] args) throws IOException {
```

Comment:
Could this have a dryrun option?
And log the requests that would be made.

Reply:
Updated to accept a case-insentive 2nd param as `update`. Without this it defaults into dry-run mode so that the user cannot accidently send data.